### PR TITLE
Move deprecated security suites to the bottom of the file

### PR DESCRIPTION
**Pull request by OR13**
Previously https://github.com/perma-id/w3id.org/pull/2175

### CODE REVIEW

File: security/.htaccess [hunk 1]
```diff
@@ -79,8 +67,19 @@ RewriteRule ^suites/secp256k1-2020/v1$ https://ns.did.ai/suites/secp256k1-2020/v
...
 RewriteRule ^suites/secp256k1-2019/v1$ https://ns.did.ai/suites/secp256k1-2019/v1 [R=302,L]
 
-
 # http://w3id.org/security/suites/x25519-2019
 
 RewriteRule ^suites/x25519-2019$ https://ns.did.ai/suites/x25519-2019 [R=302,L]
 RewriteRule ^suites/x25519-2019/v1$ https://ns.did.ai/suites/x25519-2019/v1 [R=302,L]
+
+# 🚧 DEPRECATED as of 04/29/2021 🚧
+# The content of thesse contexts is available under /suites above.
```
Comment (OR13) [inline security/.htaccess:76]:
```suggestion
# The content of these contexts is available under /suites above.
```

File: security/.htaccess [hunk 2]
```diff
@@ -79,8 +67,19 @@ RewriteRule ^suites/secp256k1-2020/v1$ https://ns.did.ai/suites/secp256k1-2020/v
 RewriteRule ^suites/secp256k1-2019$ https://ns.did.ai/suites/secp256k1-2019 [R=302,L]
 RewriteRule ^suites/secp256k1-2019/v1$ https://ns.did.ai/suites/secp256k1-2019/v1 [R=302,L]
 
-
 # http://w3id.org/security/suites/x25519-2019
 
 RewriteRule ^suites/x25519-2019$ https://ns.did.ai/suites/x25519-2019 [R=302,L]
 RewriteRule ^suites/x25519-2019/v1$ https://ns.did.ai/suites/x25519-2019/v1 [R=302,L]
+
+# 🚧 DEPRECATED as of 04/29/2021 🚧
```
Comment (davidlehn) [inline security/.htaccess:75]: This is maybe a silly complaint, but would it be ok to just use simple ASCII here?

Comment (OR13) [reply security/.htaccess:75]: the point of the emoji is to make it clear there are deprecations, is reducing the visibility of deprecation warnings an enhancement to this PR?

Certainly if there is an encoding issue that this causes, I would remove it, but if this is just aesthetic preferences, I prefer to see deprecation warnings loud and clear.

Comment (OR13) [reply security/.htaccess:75]: @davidlehn feel free to suggest changes, if lots of other folks prefer your aesthetic preferences, I am happy to accept them.

Comment (davidlehn) [reply security/.htaccess:75]: I thought CAPITALIZATION was enough. :-)  And/or just some `################` delimiters.  I do have a path through some servers that has some console unicode issues, but I'll deal with the `?`s.

Comment (TallTed) [reply security/.htaccess:75]: Keeping in mind that not all emoji are visible on all systems (e.g., the `?`s displayed in some of @davidlehn's usage), I would recommend sticking with ASCII flags in places like this.  If you really want the construction barriers, you could probably find or recreate some ASCII art that delivers them. :-)

**Pull request merged**